Demo1.js

```diff
@@ -27,6 +27,15 @@ export default React.createClass({
     };
   },
 
+  handleTouchStart: function(key, pressLocation, event) {
+    this.handleMouseDown(key, pressLocation, event.touches[0]);
```

Comment:
What I mean is, maybe instead of `handle(a, b, event.touches[0])`, `handle(a, b, firstPoint)` is clearer in conjunction with `destructuring on `event`: ({touches: [firstPoint]})`? At this point it's just a nit. IF you think the current one's clearer let's merge.